src/mlpack/methods/ann/init_rules/gaussian_init.hpp

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file gaussian_init.hpp
+ * @author Kris Singh
+ *
+ * Intialization rule for the neural networks. This simple initialization is
+ * performed by assigning a gaussian matrix with a given mean and variance
+ * to the weight matrix.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_INIT_RULES_GAUSSIAN_INIT_HPP
+#define MLPACK_METHODS_ANN_INIT_RULES_GAUSSIAN_INIT_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * This class is used to initialize weigth matrix with a gaussian.
+ */
+class GaussianInitialization
+{
+ public:
+  /**
+   * Initialize the gaussian initialization rule with mean=0 and
+   * variance = 1.
+   *
+   * @param mean The mean of the gaussian.
+   * @param variance The variance of the gaussian.
+   */
+  GaussianInitialization():
```

Comment:
To be able to specify mean and covariance, you should also implement the constructor, something like:

```
GaussianInitialization(const MatType& mean, const MatType& covariance) : ...
```